File: greenfox/1. foundation-part/data-structures/ShoppingList.java
import java.util.ArrayList;

public class ShoppingList {
	public static void main(String[] args) {
		ArrayList<String> list = new ArrayList<>();
		list.add("eggs");
		list.add("milk");
		list.add("fish");
		list.add("apples");
		list.add("bread");
		list.add("chicken");

		if (list.contains("milk")) {
			System.out.println("yes");
		} else System.out.println("no");

		if (list.contains("bananas")) {
			System.out.println("yes");
		} else System.out.println("no");
	}
}

File: greenfox/4. orientation-part/js-basics/shopping-list.js
const items = ["eggs", "milk", "fish", "apples", "bread", "chicken"];

console.log("Do we have milk in the shopping list? (yes/no)");
console.log(doesItemExist(items, "milk"));
console.log("");

console.log("Do we have bananas in the shopping list? (yes/no)");
console.log(doesItemExist(items, "bananas"));

function doesItemExist(list, param) {
	return list.includes(param);
}